docs/first-time-run.md

```diff
@@ -129,6 +129,36 @@ oc create secret generic hydra-credentials \
   --from-literal password=<password>
 ```
 
+#### Prow-kubeconfig
```

Comment:
Should we include any context here that the target prow cluster must be a `ci-operator`-enabled cluster? 
https://docs.ci.openshift.org/docs/architecture/ci-operator/

This might be perceived such that the hosted pipeline, and more specifically preflight-trigger, can target _any_ prow cluster, and I'm not sure that's the case.

I'm good either way.

Reply:
It would be nice to link to something for more details.